de.manumaticx.pagingcontrol/controllers/widget.js

```diff
@@ -188,6 +194,9 @@ function adjustePositions() {
  * @param {Ti.UI.Scrollableview} scrollable view
  */
 exports.setScrollableView = function(_sv){
+    if($.scrollableView) {
+        throw "Already initialized";
+    }
```

Comment:
setter cause multiply calls to init function. Memory leaks with event handlers on old tabs?